lib/resty/auto-ssl/ssl_certificate.lua

```diff
@@ -113,9 +113,11 @@ local function get_cert(auto_ssl_instance, domain)
   end
 
   -- Finally, issue a new certificate if one hasn't been found yet.
-  fullchain_pem, privkey_pem = issue_cert(auto_ssl_instance, storage, domain)
-  if fullchain_pem and privkey_pem then
-    return convert_to_der_and_cache(domain, fullchain_pem, privkey_pem, true)
+  if ssl_options and ssl_options["generate_certs"] ~= false then
```

Comment:
As far as I can tell, this defaults to not issuing any certificates when `ssl_options` is not there or the value is not specified. When users update to a version containing this change, their setup will stop working for new domains.

As to not break existing setups without need, I suggest to change the default for this to `true`.